vocabularies-qsi/addr-geocode-types.ttl

```diff
@@ -222,10 +172,10 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 :service-meter
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "SM"@en ;
     skos:definition "The utility meter (e.g. box, or underground chamber)."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:narrower
         :electricity-meter ,
```

Comment:
This should also include the narrower items:
water-meter
gas-meter
electricity-meter

Reply:
Same as comment above. Expand by clicking the blue section in the GitHub UI.